pages/releases.mdx

```diff
@@ -0,0 +1,149 @@
+import { Step } from '../components/Step'
+import { Callout, Steps } from "nextra/components"
+
+# Release notes
+*The latest Snaplet product updates from the Snaplet team.*
+
+
+<Steps>
```

Comment:
I'm not sure about why we use `Steps` here?

Reply:
If we want something like Neon: https://neon.tech/docs/release-notes
I'll make a component for it. 😄

Reply:
We're using Steps because I was trying to replicate how some other devtools do their release notes, but it is definitely a hacky solution. I'd love for you to make me a better component please!